src/test/java/com/google/api/generator/engine/writer/JavaWriterVisitorTest.java

```diff
@@ -380,6 +381,67 @@ public void writeMethodInvocationExpr_chained() {
         writerVisitor.write(), "libraryClient.streamBooksCallable().doAnotherThing().call()");
   }
 
+  @Test
+  public void writeAnonymousClassExpr_basic() {
+      ConcreteReference ref = ConcreteReference.withClazz(Runnable.class);
+      TypeNode type = TypeNode.withReference(ref);
+      AssignmentExpr assignmentExpr = createAssignmentExpr("foobar", "false", TypeNode.BOOLEAN);
+      ExprStatement statement = ExprStatement.withExpr(assignmentExpr);
+      MethodDefinition method =
+          MethodDefinition.builder()
+              .setScope(ScopeNode.PUBLIC)
+              .setReturnType(TypeNode.VOID)
+              .setName("run")
+              .setIsOverride(true)
+              .setBody(Arrays.asList(statement))
+              .build();
+  
+      AnonymousClassExpr anonymousClassExpr =
+          AnonymousClassExpr.builder().setType(type).setMethods(Arrays.asList(method)).build();
+      anonymousClassExpr.accept(writerVisitor);
+      assertEquals(writerVisitor.write(), "new Runnable() {\n@Override\npublic void run() {\nboolean foobar = false;\n}\n}");
```

Comment:
Can we separate out the expected code string into distinct lines, so it's easier to read?